dts/arm/st/h7/stm32h7a3.dtsi

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2021 Electrolance Solutions
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/h7/stm32h7.dtsi>
+
+/ {
+	soc {
+		flash-controller@52002000 {
+			flash0: flash@8000000 {
+				write-block-size = <32>;
+				erase-block-size = <DT_SIZE_K(128)>;
+			};
+		};
+
+		dmamux1: dmamux@40020800 {
+			dma-requests= <107>;
```

Comment:
Actually, the dma-request gives the max DMAREQ_ID[6:0] --> upper limit of DMAMUX1 req inputs table. Here, more than 107 + 8 generator req. entries are available. 
Setting dma-request to 107 plus adding the 8 dmamux generator req gives a lower value to the max possible value of the table. Even some (ex. 113, 114, 115 ) are reserved, other values like 116 to 119 are still  possible. The DMAREQ_ID[6:0] of the DMAMUX1_CxCR is a value from 1 to 119. --> considering 8 dmamux generator entries, the dma-requests is 111.

Reply:
Do you mean also reserved are excluded?

![image](https://user-images.githubusercontent.com/17961318/149606796-fc7a51c1-f14b-4201-ba65-73d56049fcdb.png)


I can see from the datasheet that the total mux input is `127`, reserved are `16`, which means the max possible ID entry is `111`. I wanted to confirm I am right, and please correct me if anything is wrong. Thanks, and I appreciate your review and feedback.